package.json

```diff
@@ -8,6 +8,7 @@
     "check-types": "tsc",
     "preview": "vite preview",
     "lint": "eslint --max-warnings 0 'src/**/*.ts' 'src/**/*.tsx' && knip",
+    "lighthouse": "SCRIVITO_TENANT=d0a154d76edf2a7bd991fc658e700a1d SCRIVITO_ORIGIN=http://localhost:8080 PRERENDER_OBJ_ID=c2a0aab78be05a4e npm run prerender && precompress --include '(**).(css|html|js)' --types br dist-ssr && for run in $(seq 6); do lhci autorun && break; done",
```

Comment:
Do we need to hard-code the `SCRIVITO_TENANT` ID? Why not take the tenant from the env? This way the lighthouse test would be automatically transferred.

Reply:
The current CI env has no `SCRIVITO_TENANT` set. The consequence of moving the setting to the CI is that we force forks with CI integration to set the env too. If this is fine, yes we could do this.